<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manager</class>
 <widget class="QMainWindow" name="manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1237</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>941</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>20</y>
      <width>231</width>
      <height>343</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="3">
        <widget class="QLineEdit" name="QueryOrder2"/>
       </item>
       <item row="2" column="0">
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>BOOKID</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CATEGORY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NAME</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PRESS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YEAR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AUTHOR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PRICE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COLLECTION</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>STOCK</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>查询条件</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="QueryOrder1"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>排列顺序</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="BookQuery">
       <property name="text">
        <string>查询图书</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>书号</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>类别</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>书名</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>出版社</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>年份</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>作者</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>价格</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>数量</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="LINEBOOKID"/>
           </item>
           <item>
            <widget class="QLineEdit" name="LINECATEGORY"/>
           </item>
           <item>
            <widget class="QLineEdit" name="LINENAME"/>
           </item>
           <item>
            <widget class="QLineEdit" name="LINEPRESS"/>
           </item>
           <item>
            <widget class="QLineEdit" name="LINEYEAR"/>
           </item>
           <item>
            <widget class="QLineEdit" name="LINEAUTHOR"/>
           </item>
           <item>
            <widget class="QLineEdit" name="LINEPRICE"/>
           </item>
           <item>
            <widget class="QLineEdit" name="LINENUMBER"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="addOneBook">
         <property name="text">
          <string>单本入库</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AddBookFromFile">
         <property name="text">
          <string>批量入库</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>图书查询结果：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>借书情况显示：</string>
    </property>
   </widget>
   <widget class="QTableView" name="Borrow">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>941</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>410</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>借书卡号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LineCardID"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="QueryCardID">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DeleteCard">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>借书书号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LineBookID"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="BorrowBook">
       <property name="text">
        <string>借书</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ReturnBook">
       <property name="text">
        <string>还书</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>570</y>
      <width>171</width>
      <height>107</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>姓名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>单位</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>类别</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLineEdit" name="LineName"/>
         </item>
         <item>
          <widget class="QLineEdit" name="LineUnit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="LineType"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="AddCard">
       <property name="text">
        <string>增加</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1237</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
